### PR TITLE
add qgss2023 lab 5

### PR DESCRIPTION
**Pull request by kevinsung**

### CODE REVIEW

File: qc_grader/challenges/qgss_2023/lab5.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from typeguard import typechecked
+
+from qc_grader.grader.grade import grade
+
+_challenge_id = "qgss_2023"
+
+
+@typechecked
+def grade_lab5_ex1(noise_model: NoiseModel) -> None:
+    grade(noise_model, "ex5-1", _challenge_id, byte_string=True)
```
Comment (vabarbosa) [inline qc_grader/challenges/qgss_2023/lab5.py:11]: this should not be necessary:

```suggestion
    grade(noise_model, "ex5-1", _challenge_id)
```

File: qc_grader/challenges/qgss_2023/lab5.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+
+@typechecked
+def grade_lab5_ex2(new_shots: int) -> None:
+    grade(new_shots, "ex5-2", _challenge_id, byte_string=True)
```
Comment (vabarbosa) [inline qc_grader/challenges/qgss_2023/lab5.py:16]: this is not needed here:

```suggestion
    grade(new_shots, "ex5-2", _challenge_id)
```

File: qc_grader/challenges/qgss_2023/lab5.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+
+@typechecked
+def grade_lab5_ex3(noise_model: NoiseModel) -> None:
+    grade(noise_model, "ex5-3", _challenge_id, byte_string=True)
```
Comment (vabarbosa) [inline qc_grader/challenges/qgss_2023/lab5.py:21]: this should also not be necessary

```suggestion
    grade(noise_model, "ex5-3", _challenge_id)
```

**Review by vabarbosa — CHANGES_REQUESTED** [review]
@kevinsung thank you! just a couple minor comments

File: qc_grader/challenges/qgss_2023/lab5.py (hunk 1 above)
Comment (kevinsung) [reply qc_grader/challenges/qgss_2023/lab5.py]: done

File: qc_grader/challenges/qgss_2023/lab5.py (hunk 2 above)
Comment (kevinsung) [reply qc_grader/challenges/qgss_2023/lab5.py]: done

**Review by vabarbosa — APPROVED** [review]
thank you!

**Pull request merged**